phoonnx/model_manager.py

```
@@ -0,0 +1,293 @@
+    @property
+    def engine(self) -> Engine:
+        return self.config.engine
+
+    @property
+    def voice_path(self) -> Path:
+        return xdg_cache_home() / "phoonnx" / "voices" / self.voice_id
+
+    def download_config(self):
+        config_path = self.voice_path / "model.json"
+        if not config_path.is_file():
+            r = requests.get(self.config_url)
```

Ruff (0.14.0)

Probable use of `requests` call without timeout

(S113)